src/python/tests/core/crash_analysis/stack_parsing/stack_analyzer_data/hwasan_allocation_tail_overwritten.txt

```diff
@@ -0,0 +1,44 @@
+==11246==ERROR: HWAddressSanitizer: allocation-tail-overwritten; heap object [0x00444b02b180,0x00444b02b182) of size 2
```

Comment:
Would it make sense to parse `0x00444b02b180` as crash address?

Reply:
Done!